test/e2e/vmagent_test.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"crypto/sha1" // #nosec G505: not used for security purposes
+	"encoding/hex"
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	"antrea.io/antrea/pkg/features"
+	. "antrea.io/antrea/test/e2e/utils"
+)
+
+const (
+	interfaceNameLength = 5
+	nameSpace           = "vm-ns"
```

Comment:
```suggestion
	namespace           = "vm-ns"
```

Reply:
Based on previous review comments, I changed it from `namespace` to `nameSpace`

Reply:
which comment? I think namespace is one word, not two?

Reply:
If you think, it should be `namespace`, I will change it back.

Reply:
@tnqn: Updated the PR to use namespace.